sass/main.scss

```diff
@@ -680,121 +680,304 @@ nav {
-            .event:nth-child(3n) {
-                border-top: 10px solid $red;
+            p {
+                color: $dark-blue;
             }
-            .event:nth-child(3n + 1) {
-                border-top: 10px solid $orange;
+        }
+
+        .schedule-tab-active {
+            transform: translateY(-5px);
+            height: 55px;
+            animation: none;
+
+            &:hover {
+                animation: none;
             }
-            .event:nth-child(3n + 2) {
-                border-top: 10px solid $light-gray;
+        }
+
+        // Specific tab styling
+        .pre-event {
+            background-color: $red;
+        }
+
+        .feb-20 {
+            background-color: $orange;
+        }
+
+        .feb-21 {
+            background-color: $light-gray;
+        }
+    }
+
+    #schedule-content-wrapper {
+        #feb-20-content, #feb-21-content {
+            display: none; // Hide the two other events by default
+
+            // A little less than the style put just below the #schedule-content-wrapper
+            .schedule-content {
+                padding-bottom: 20px;
             }
 
-            .event {
-                background: white;
-                border-radius: 15px;
-                margin: 10px 0px;
-                padding: 10px;
-                position: relative;
+            .show-full-schedule, .hide-full-schedule {
+                bottom: 0;
+                text-align: right;
+                margin-right: 20%; // TODO: make more aligned with right of events
+                padding-bottom: 40px;
+                display: flex;
+                justify-content: flex-end;
+                align-items: center;
+                cursor: pointer;
 
-                &.past {
-                    background-color: #bfbfbf;
+                p {
+                    font-size: 1.3em;
+                    user-select: none;
                 }
 
-                .time {
-                    color: $light-blue;
+                i {
+                    color: $red;
+                    font-size: 2em;
+                    margin-left: 10px;
                 }
+            }
 
-                .title {
-                    color: black;
-                    font-size: 1.3em;
-                }
+            .hide-full-schedule {
+                display: none;
+            }
+
+            .events {
+                height: 370px;
+                padding-bottom: 10px;
+                overflow: hidden;
+            }
+        }
+    }
+
+    .schedule-content {
+        display: flex;
+        justify-content: space-around;
+        padding: 60px 100px;
+    }
 
-                .marker {
-                    $marker-width: 20px;
+    // Normal events
+    .days {
+        width: 40%;
+        display: flex;
+        justify-content: flex-start;
+        flex-direction: column;
+        user-select: none;
+        cursor: pointer;
+        color: white;
+    }
+
+    .day {
```

Comment:
Confusing class names. Maybe put "date" or "day-label" for this or "event" or "day-box" for .days

Reply:
good call!

Reply:
#1161